src/business-logic/services/test/rundown-timeline-service.spec.ts

```diff
@@ -55,6 +56,22 @@ describe(RundownTimelineService.name, () => {
 
       await expect(() => testee.deleteRundown(rundown.id)).rejects.toThrow(ActiveRundownException)
     })
+
+    it('throws an exception, when trying to active a rundown when there is another already activated rundown', async () => {
+      const mockRundownRepository: RundownRepository = mock<RundownRepository>()
+
+      const activeBasicRundown: Rundown = EntityMockFactory.createRundown({ isRundownActive: true })
+
+      const basicRundownArray: Rundown[] = [activeBasicRundown];
+
+      const rundownToActivate: Rundown = EntityMockFactory.createRundown({isRundownActive: false})
+
+      when(mockRundownRepository.getBasicRundowns()).thenResolve(basicRundownArray)
+
+      const testee: RundownTimelineService = createTestee({rundownRepository: instance(mockRundownRepository)})
+
+      await expect(() => testee.activateRundown(rundownToActivate.name)).rejects.toThrow(AlreadyActivatedException)
```

Comment:
You should use `rundownToActivate.id`, not `rundownToActivate.name` since `activateRundown()` expects a Rundown id. 
Both id and name are strings, so that's why you don't get an error, but you should still give it the id :)

Reply:
changes made